lib/helpers/handleMouseButtonProtocol.js

```diff
@@ -15,7 +23,32 @@ let handleMouseButtonProtocol = function (requestPath, button) {
     return this.requestHandler.create(
         requestPath,
         { button: button }
-    )
+    ).catch(err => {
+        /**
+         * jsonwire command not supported try webdriver endpoint
+         */
+        if (ProtocolError(err)) {
```

Comment:
This will be always true, no?

Reply:
Well, isn't this only true of a "ProtoclError", so if there is another error type we won't enter in here?  Is that why?

Reply:
Good catch, this should actually be checking for unknown command.

Reply:
@christian-bromann @pmerwin  Do these changes look good?